src/settings.rs

```diff
@@ -1,20 +1,55 @@
+use crate::cli::Opts;
 use config::{Config, ConfigError, Environment, File};
 use serde::Deserialize;
 use std::env;
 
+#[derive(Debug, Deserialize)]
+pub struct Peers {
+    /// later a list of urls or ips that the application can use to discover peers
+    pub source: String,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Data {
+    /// Local path for sled data store
+    pub path: String,
```

Comment:
I changed the name of these so they settable from the environment variable step. I could get `APP_DATA_PATH` Env Var to set it but not `APP_DATA_DATA_PATH`